apps/sso/components/login-card/login-card.tsx

```diff
@@ -137,3 +137,22 @@ const StyledErrorText = styled(Text)`
   padding-top: 1rem;
   color: red;
 `;
+const Input = styled.input`
+  border: solid 0.1rem #bcbcbc;
```

Comment:
```suggestion
  border: 0.1rem solid #bcbcbc;
```
Border syntax is wrong I think